### PR TITLE
Rollershutter state input

### PR DESCRIPTION
**Pull request by aounhaider1**
fixes https://github.com/eclipse/smarthome/issues/2560

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js [hunk 1]
```diff
@@ -527,6 +527,14 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     if ($scope.item.state === 'UNDEF' || $scope.item.state === 'NULL') {
         $scope.item.state = '-';
     }
+    $scope.sendStateCommand = function(command) {
+        $scope.sendCommand(command);
+        if (command == "DOWN") {
+            $scope.item.state = 100;
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: Hm, are you manually setting the state here? You should not - the runtime should send according events, which should be used to update the displayed status.

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html [hunk 2]
```diff
@@ -129,24 +129,31 @@ <h2 class="state edit" ng-if="!editMode" ng-click="editState()">{{getStateText(i
...
 											<div ng-switch-when="Rollershutter" ng-controller="RollershutterItemController">
-												<div layout="row" layout-align="start center">
+												<div layout="row" layout-align="start center" flex="100">
 													<div flex="40" layout="row" layout-align="start center">
 														<p>
 															<i class="material-icons">{{getIcon(item.category, 'format_align_justify')}}</i>{{getLabel(item.category, item.label, 'Position')}}
 														</p>
 													</div>
+													<div flex="60" layout="row" layout-align="end center">
+                                                        <div ng-show="isReadOnly(item)">
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html]: something's messed up with the formatting here.

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js (hunk 1 above)
Comment (aounhaider1) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: This is a click handler and I am just setting the state before sending the command to the runtime. Its being done the same way in all other control widgets.

**Comment by aounhaider1** [comment]
Formatted control.html

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js (hunk 1 above)
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js]: Correct. Depending on the real hardware, you might get multiple updates with incremental changes.

**Comment by aounhaider1** [comment]
Updated.

**Pull request merged**